woocommerce/payment-gateway/admin/class-sv-wc-payment-gateway-admin-order.php

```diff
@@ -197,7 +217,9 @@ public function process_capture_charge_bulk_order_action() {
 
 				$order = wc_get_order( $order_id );
 
-				$this->maybe_capture_charge( $order );
+				if ( $order && $gateway = $this->get_order_gateway( $order ) ) {
```

Comment:
We don't have a rule in our code style guide about this.

There are some sources that discourage having a variable assignment in condition. It may be confusing or prone to errors (maybe you want to check if a variable exists or is of a type and write `=` instead of `==` and vice versa.

I personally don't think it's an issue, when you have 1-2 conditions only. If there's more than 2 maybe it's best to break down the `if` statement into multiple statements.

However, regardless of this, maybe it's a good idea to write the condition where you also assign a variable in parenthesis like so:

```php
if ( $order && ( $gateway = $this->get_order_gateway( $order ) ) ) {
	// ...
}
```

This should highlight that you're assigning (or overwriting) a variable. I personally do this, but we can discuss in WC dev meeting what's best here, or leave it to personal preferencee.